src/components/Editor/Configs/BackgroundConfig.tsx

```diff
@@ -44,28 +46,31 @@ const BackgroundConfig: FC<
         <FormLabel>Fill</FormLabel>
         <SimpleGrid columns={5} gap={4}>
           {GRADIENT_COLORS.map((c) => (
```

Comment:
instead of c it would be great to mention the full variable name, it will increase readability